hydra/core/override_parser/overrides_visitor.py

```diff
@@ -309,6 +308,55 @@ def _createPrimitive(
                 return node.getText()  # type: ignore
         return ret
 
+    def _unescape_quoted_string(self, text: str) -> str:
```

Comment:
Please add roundtrip testing for _unescape_quoted_string and with_quotes similar to what is in my code snippet.

Reply:
As just discussed, this is done in the current test => resolving